### PR TITLE
Fix for #6158

### PR DESCRIPTION
**Pull request by anneomcl**
Converting Vector2 to Size2 for scaling functions

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -9,6 +9,7 @@ import sys
 import methods
 
 methods.update_version()
+ANGLE_SRC_PATH = "C:\Users\anneo\OneDrive\Documents"
```
Comment (akien-mga) [inline SConstruct]: Could you remove this part? It obviously wasn't meant to be committed :)

**Review by akien-mga — CHANGES_REQUESTED** [review]
Looks good to me apart from the `SConstruct` change.

File: SConstruct (hunk 1 above)
Comment (anneomcl) [inline SConstruct]: Yup, sorry about that!

**Comment by akien-mga** [comment]
Thanks. Could you squash the two commits together so that the SConscript history is not altered?
```
git rebase -i HEAD~2
// replace "pick" on second line (second commit) with "fixup"
// save and quit the text editor, it will rebase
git log // check that it worked
git show // same
// if all good:
git push --force
```

**Review by akien-mga — APPROVED** [review]

**Comment by anneomcl** [comment]
Should be good now, thank you! Getting used to the Git workflow again.

**Pull request merged**